src/reference/asciidoc/index-single.adoc

```diff
@@ -40,6 +40,7 @@ include::./system-management.adoc[]
 include::./endpoint-summary.adoc[]
 
 include::./amqp.adoc[]
+include::./cassandra.adoc[]
```

Comment:
```suggestion
include::./amqp.adoc[]

include::./cassandra.adoc[]
```

A blank line is needed between includes.

Reply:
Do you have a confirmation that it still doesn't work without blank lines in between?
May be a new version is much robust already?
How to check it locally to be sure that nothing has broken?
Thanks